### DISCUSSION TITLE
Welcome to dataprep Discussions!

### DISCUSSION THREAD

**Discussion by dovahcrow** [sfu-db/dataprep#550] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ManjunathSrinivas** [comment]
Hi ,

One question on Data prep,  Create_report function, Is there any way to control width of data being displayed.

In my case I have very long column names, In the Overview I am unable see names

**Comment by jinglinpeng** [comment]
Hi @ManjunathSrinivas , are you talking about the column name in overview-insight?
![image](https://user-images.githubusercontent.com/18078770/136740018-b04b3b15-f810-4927-bb3f-55cdefff740e.png)
Currently we just cut the long column name, since showing the full name will affect the layout.
